Backend/Bankbackend2/WeatherForecast.cs

```diff
@@ -0,0 +1,13 @@
+namespace Bankbackend2
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+
+        public int TemperatureC { get; set; }
+
+        public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
+
+        public string? Summary { get; set; }
+    }
+}
```

Comment:
Warum hast du hier was für Wettervorhersagen?

Reply:
das war noch als standart drinne wenn man die Api eröffnet ich hab es als vergleichs wert benutzt

Reply:
Könntest du alles was du nicht benutzt entfernen oder falls du die dateien als vergleich noch brauchst in deine .gitignore packen? Das macht für mich alles übersichtlicher und einfacher zu verstehen

Reply:
jo mach ich